crypto.c

```diff
@@ -338,7 +332,7 @@ hmac_md5(unsigned char *text, int text_len, unsigned char *key, int key_len,
         MD5_Update(&context, k_ipad, 64);     /* start with inner pad */
         MD5_Update(&context, text, text_len); /* then text of datagram */
 
-	MD5_Final(digest, &context);          /* finish up 1st pass */
+        MD5_Final(digest, &context);          /* finish up 1st pass */
```

Comment:
just unintentional whitespace change?

Reply:
Yes. The whitespace is a bit funky in that function (it doesn't follow style(9) -- uses 8 space indents).

Reply:
I reverted the whitespace change.